Gemfile.lock

```diff
@@ -94,9 +96,10 @@ DEPENDENCIES
   tzinfo
   tzinfo-data
   wdm (>= 0.1.0)
+  webrick (~> 1.7)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 3.0.3p157
 
 BUNDLED WITH
-   2.3.10
+   2.3.5
```

Comment:
And this 2.3.5

Reply:
2.3.10 or .5?

Reply:
To .10, preferably just remove any major changes to this file.

Reply:
```suggestion
   2.3.10
```